registry/server/templates/services/templatesRepository.ts

```diff
@@ -182,6 +194,28 @@ export class TemplatesRepository {
         return { type: 'ok', template: updatedTemplate };
     }
 
+    async partialUpdateTemplate(
+        templateName: string,
+        payload: PartialUpdateTemplatePayload,
+        user: User | undefined,
+    ): Promise<PartialUpdateTemplateResult> {
+        const templateExists = await this.db(Tables.Templates).where({ name: templateName }).select(1).first();
```

Comment:
doesnt `db` have some firm syntax for `findOne` ?

Reply:
No, it doesn't. This is the optimal syntax for this kind of queries.

Why `first()`? Because I need either a row or `null`.
Why `select(1)`? Because I don't need to transfer any data at all.